### DISCUSSION TITLE
Katas 7,8,9 y 10

### DISCUSSION THREAD

**Discussion by RicardoUValencia** [LaunchX-InnovaccionVirtual/CursoIntroPython#103] [Q&A]
Hola alguien sabe cuando se habilitaran las katas que siguen despues de la kata 6? Es que en Teams tiene la fecha de hoy 10 de Febrero de 2022 que estaran disponibles.

**Comment by AzaelBarbosa** [comment]
Hola, las habilitan hoy por la noche, pero dijeron que no todas, quizas solo 2 o 3

**Comment by ajcarrillo** [comment]
Hola, date una vuelta por la discusión #98
